code/modules/mob/living/basic/slime/slime.dm

```diff
@@ -250,29 +250,23 @@
 		. += span_warning("It seems too overcroweded to properly reproduce!")
 
 ///Changes the slime's current life state
-/mob/living/basic/slime/proc/set_life_stage(new_life_stage = SLIME_LIFE_STAGE_BABY)
+/mob/living/basic/slime/proc/set_life_stage(new_life_stage = SLIME_LIFE_STAGE_BABY, initial = FALSE)
 	life_stage = new_life_stage
-
-	switch(life_stage)
-		if(SLIME_LIFE_STAGE_BABY)
-
-			health = initial(health)
-			maxHealth = initial(maxHealth)
-
-			obj_damage = initial(obj_damage)
-			melee_damage_lower = initial(melee_damage_lower)
-			melee_damage_upper = initial(melee_damage_upper)
-			wound_bonus = initial(wound_bonus)
-
-		if(SLIME_LIFE_STAGE_ADULT)
-
-			health = 200
-			maxHealth = 200
-
-			obj_damage = 15
-			melee_damage_lower += 10
-			melee_damage_upper += 10
-			wound_bonus = -90
+	if(life_stage == SLIME_LIFE_STAGE_ADULT)
+		health /= 0.75
+		maxHealth /= 0.75
+		melee_damage_lower *= 2
+		melee_damage_upper *= 2
+		obj_damage = 15
+		wound_bonus = -90
+
+	else if(!initial)
+		health *= 0.75
+		maxHealth *= 0.75
+		melee_damage_lower *= 0.5
+		melee_damage_upper *= 0.5
+		obj_damage = initial(obj_damage)
+		wound_bonus = initial(wound_bonus)
```

Comment:
Any reason you still initial() these as opposed to modifying them like you do with the rest?

Reply:
i don't think anything is in the game that would change them (including fugu), so kept them at initial to simplify it.